packages/builders/scripts/fast-usdc/init-fast-usdc.js

```diff
@@ -122,7 +122,15 @@ export default async (homeP, endowments) => {
       return configurations[net].feedPolicy;
     }
     if (!feedPolicy) throw Error(feedPolicyUsage);
-    return JSON.parse(feedPolicy);
+    const parsed = JSON.parse(feedPolicy);
+    if (!parsed.chainPolicies) {
+      // chainPolicies contain bigints which can't be stringified
```

Comment:
this should use LegibleCapData then, I think

Reply:
oic... this is saying "if the CLI arg doesn't have chainPolicies, fill it in from MAINNET"
maybe that's good enough